shared/settings/landing/index.desktop.js

```diff
@@ -294,26 +294,47 @@ function AccountPassphrase({onChangePassphrase}: {onChangePassphrase: () => void
   )
 }
 
+function AccountFirstPassphrase({onChangePassphrase}: {onChangePassphrase: () => void}) {
+  return (
+    <Box style={{...globalStyles.flexBoxRow, alignItems: 'center', minHeight: ROW_HEIGHT}}>
+      <Text type="Body" style={{marginRight: globalMargins.xtiny}}>
+        Passphrase:
+      </Text>
+      <Text type="Body">
+        Not set! You should{' '}
+        <Text type="Body" style={{color: globalColors.blue}} link={true} onClick={onChangePassphrase}>
+          set a passphrase
+        </Text>
+        .
+      </Text>
+    </Box>
+  )
+}
+
 function Account({
   email,
   isVerified,
   onChangeEmail,
   onChangePassphrase,
   onChangeRememberPassphrase,
   rememberPassphrase,
+  hasRandomPW,
 }: AccountProps) {
+  const Passphrase = !hasRandomPW ? AccountPassphrase : AccountFirstPassphrase
   return (
     <Box style={{...globalStyles.flexBoxColumn, marginBottom: globalMargins.medium}}>
       <AccountEmail email={email} isVerified={isVerified} onChangeEmail={onChangeEmail} />
       <Divider />
-      <AccountPassphrase onChangePassphrase={onChangePassphrase} />
-      <Divider />
-      <Checkbox
-        checked={rememberPassphrase}
-        label="Remember my passphrase"
-        onCheck={onChangeRememberPassphrase}
-        style={{paddingTop: globalMargins.small}}
-      />
+      <Passphrase onChangePassphrase={onChangePassphrase} />
+      {!hasRandomPW && <Divider />}
```

Comment:
You can do this and the `Checkbox` together by surrounding them with `<> ... </>`
```
{!hasRandomPW && (<><Thing1 /><Thing2 /></>)}`

Reply:
Also, do you care whether it's `false` or `null` here?

Reply:
if it's `null` it should assume that user has passphrase, so none of the special NOPW treatment should happen. I think it's fine if it refreshes while user is in there.